testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -617,4 +617,32 @@ public void testSetViewAuthorizationWithSecurityInvoker()
     {
         assertQuerySucceeds("ALTER VIEW mock.default.test_view_invoker SET AUTHORIZATION some_other_user");
     }
+
+    @Test
+    public void testUseStatementAccessControl()
+    {
+        executeExclusively(() -> {
```

Comment:
I have a feeling that `executeExclusively` is overused in this class. If we're using entities with random names, and make sure that the deny rules in the `TestingAccessControlManager` only reference these random names, then they should be able to run concurrently. That's out of scope for this change, though.

Reply:
Here I also change the behavior via `TestingAccessControlManager` so this `executeExclusively` prevent to break other tests